src/GraphQL/Extensions/GraphQLBuilderExtensions.cs

```diff
@@ -1168,5 +1168,28 @@ public static IGraphQLBuilder AddExecutionStrategy<TExecutionStrategy>(this IGra
             return builder;
         }
         #endregion
+
+        #region - UseTelemetry -
+#if NET5_0_OR_GREATER
```

Comment:
Another option - move all OTel code under ifdefs into new project and use net5 tfm there.

Reply:
Please see [previous comment](https://github.com/graphql-dotnet/graphql-dotnet/pull/3467#discussion_r1092538553).